File: u3u2/Assets/Scripts/app/net/c2s/CGIosAndroidCharge.cs
using System;
using System.IO;
namespace app.net
{

/**
 * IOS和android直冲查询
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGIosAndroidCharge :BaseMessage
{
	
	
	public CGIosAndroidCharge ()
	{
	}
	
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_IOS_ANDROID_CHARGE;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}

File: IdeaServer/zlj2019/game_server/src/com/imop/lj/gameserver/player/msg/CGIosAndroidCharge.java
package com.imop.lj.gameserver.player.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.player.handler.PlayerHandlerFactory;

/**
 * IOS和android直冲查询
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGIosAndroidCharge extends CGMessage{
	
	
	public CGIosAndroidCharge (){
	}
	
	
	@Override
	protected boolean readImpl() {


		return true;
	}
	
	@Override
	protected boolean writeImpl() {

		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_IOS_ANDROID_CHARGE;
	}
	
	@Override
	public String getTypeName() {
		return "CG_IOS_ANDROID_CHARGE";
	}


	@Override
	public void execute() {
		PlayerHandlerFactory.getHandler().handleIosAndroidCharge(this.getSession().getPlayer(), this);
	}
}